extensions/widgets/paletteactions/paletteactions.lcb

```diff
@@ -700,8 +700,15 @@ private handler drawActions()
         put element tActionIndex of mActionRects into tRect
 
         variable tMode as String
+        variable tEnabled as optional Boolean
+        // tAction["enabled"] could be a string or boolean
```

Comment:
@livecodeali @runrevmark this all seems like too much work. It seems to me it would be nice if we had the equivalent of `parse any as <type>` which followed the rules used by LCS for `TryToEval`